StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/ConsumerSession/ConsumerSessionModels.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 struct ConsumerSessionData: Decodable {
     let clientSecret: String
     let emailAddress: String
-    let redactedPhoneNumber: String
+    let redactedFormattedPhoneNumber: String
```

Comment:
[backend code that verifies that this should be non-null to avoid any decoding issues](https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/api/networking/resources/consumer_session_resource.rb#L18)